src/cromshell/status/command.py

```diff
@@ -32,19 +32,47 @@ def main(config, workflow_id):
     # Check if Cromwell Server Backend works
     http_utils.assert_can_communicate_with_server(config)
 
+    requested_status_json, workflow_status, updated_ret_val = query_status(
+        config, workflow_id
+    )
+    ret_val = updated_ret_val
+
+    # Display status to user:
+    line_string = requested_status_json
```

Comment:
why rename this here?  Lets just print the requested_status_json after replacement.

Reply:
+1
Plus It isn't [good practice](https://docs.quantifiedcode.com/python-anti-patterns/readability/putting_type_information_in_a_variable_name.html) to use type suffix when naming variables

Reply:
Push back a little: I didn't change this as it was in the original code. I used pycharm to refactor out a function. 
Regardless, I'll update this PR with the requested changes, and move this to utils.